kv/kvserver: TestClosedTimestampFrozenAfterSubsumption failed

kv/kvserver.TestClosedTimestampFrozenAfterSubsumption [failed](https://mesolite.cluster.engflow.com/invocations/default/729d621f-e501-4bfc-bd62-0c724a9b3469?testReportRun=4&testReportShard=7&testReportAttempt=1#targets-Ly9wa2cva3Yva3ZzZXJ2ZXI6a3ZzZXJ2ZXJfdGVzdA==) on release-25.4 @ [7d7b1725f18aa5d98bdb9b09f170cad0a94f931c](https://github.com/cockroachdb/cockroach/commits/7d7b1725f18aa5d98bdb9b09f170cad0a94f931c):


```
=== RUN   TestClosedTimestampFrozenAfterSubsumption
    test_log_scope.go:171: test logs captured to: outputs.zip/logTestClosedTimestampFrozenAfterSubsumption2497113972
    test_log_scope.go:82: use -show-logs to present logs inline
    closed_timestamp_test.go:855: -- test log scope end --
test logs left over in: outputs.zip/logTestClosedTimestampFrozenAfterSubsumption2497113972
--- FAIL: TestClosedTimestampFrozenAfterSubsumption (100.70s)
=== RUN   TestClosedTimestampFrozenAfterSubsumption/rhs_lease_transfer_while_subsumed
    testcluster.go:1437: condition failed to evaluate within 1m30s: from testcluster.go:1439: leader hasn't stepped down yet
--- FAIL: TestClosedTimestampFrozenAfterSubsumption/rhs_lease_transfer_while_subsumed (96.45s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=4</code>
 - <code>shard=7</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #160178 kv/kvserver: TestClosedTimestampFrozenAfterSubsumption failed [C-test-failure O-robot T-kv branch-release-26.1 release-blocker s390x-test-failure]
</p>
</details>
/cc @cockroachdb/kv-triage
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestClosedTimestampFrozenAfterSubsumption.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-58212

--- a/pkg/kv/kvserver/closed_timestamp_test.go
+++ b/pkg/kv/kvserver/closed_timestamp_test.go
@@ -736,13 +736,13 @@ func TestClosedTimestampFrozenAfterSubsumption(t *testing.T) {
 			//
 			// NB: We want to ensure that the closed timestamp on the LHS is less than
 			// the lease start time for the RHS after the lease transfer. To ensure
-			// this holds and doesn't flake, we increase the target duration to 10
+			// this holds and doesn't flake, we increase the target duration to 30
 			// seconds.
 			tc, _, _ := setupClusterForClosedTSTesting(ctx, t, 5*time.Second, 100*time.Millisecond, clusterArgs, "cttest", "kv")
 			defer tc.Stopper().Stop(ctx)
 			sqlDB := sqlutils.MakeSQLRunner(tc.ServerConn(0))
 			sqlDB.ExecMultiple(t, strings.Split(`
-SET CLUSTER SETTING kv.closed_timestamp.target_duration = '10s';
+SET CLUSTER SETTING kv.closed_timestamp.target_duration = '30s';
 SET CLUSTER SETTING kv.closed_timestamp.side_transport_interval = '100ms';
 SET CLUSTER SETTING kv.rangefeed.closed_timestamp_refresh_interval = '100ms';
 SET CLUSTER SETTING kv.closed_timestamp.follower_reads.enabled = true;

